aea/configurations/base.py

```diff
@@ -636,6 +636,7 @@ class PackageConfiguration(Configuration, ABC):
     """
 
     default_configuration_filename: str
+    package_type: PackageType
```

Comment:
put it as class attribute so it can be retrieved in the loader just from the Configuration class.